plugins/slo/pkg/slo/slo_test.go

```diff
@@ -586,13 +606,14 @@ var _ = Describe("Converting SLO information to Cortex rules", Ordered, Label(te
 		Specify("The individual parts of the raw SLI queries should return data from cortex", func() {
 			// needed to ensure that prometheus agent registers a 200 status code
 			// otherwise the test is flaky
-			time.Sleep(time.Second)
 			rawGood, err := sloObj.RawGoodEventsQuery("5m")
 			Expect(err).NotTo(HaveOccurred())
 			rawTotal, err := sloObj.RawTotalEventsQuery("5m")
 			Expect(err).NotTo(HaveOccurred())
-
-			respGood, err := adminClient.Query(ctx, &cortexadmin.QueryRequest{
+			Eventually(func() error {
+				return nil
+			}, time.Second*90, time.Second*30).Should(BeNil())
```

Comment:
this seems extremely long

Reply:
it has to do with recording rule evaluation times, these tests are not run in CI